.gitignore

```diff
@@ -2,6 +2,8 @@
 *.pico
 *.dSYM
 *.exe
+*.dll
+*.obj
```

Comment:
There are "hfile_*.dll", "cyg*.dll", and "lib*.dll" patterns but nothing to catch pthreadVC2.dll, the pthreads library for Windows. I don't see why one would ever want to check in dll's in git, so I suggest that we keep the more generic *.dll pattern instead of the more specific ones.